core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -73,6 +77,21 @@
     UNKNOWN,
   }
 
+  /** Use to filter property keys that start with alluxio.*/
+  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
+  /** Use to filter property keys that start with aws.*/
+  private static final String AWS_CONF_PREFIX = "aws";
+  /** Use to filter property keys that start with fs.*/
+  private static final String FS_CONF_PREFIX = "fs";
+  /** Prefixes that are not related to workers but exists in worker-related scope.*/
+  private static final String[] WORKER_NOT_RELATED_PREFIX = {"alluxio.fuse", "alluxio.proxy",
```

Comment:
fuse is Scope.NONE, proxy is Scope.NONE, integration is scope.NONE

Reply:
Many of the Fuse, Proxy, and Integration properties have the ConsistencyCheckLevel = WARN and even ENFORCE(like "alluxio.integration.mesos.alluxio.jar.url", ""alluxio.integration.mesos.master.name")

If we set them as Scope.NONE, then we will not check those properties at all(both in masters and workers).

Reply:
Good call, for the properties I mentioned that do not apply to Clients, Workers, or Masters, we might as well have `ConsistencyCheckLevel.IGNORE`